tasks/main.yml

```diff
@@ -138,16 +200,15 @@
 
 - name: PKCS#12 certs
   include_tasks: pkcs12_cert.yml
-  when: ((eduroam.radsec is defined) and (eduroam.radsec.certificate is undefined)) and
-        ((eduroam.EAP is defined) and (eduroam.EAP.certificate is undefined))
+  when: eduroam.radsec is undefined or eduroam.EAP is undefined
 
 - name: PEM certs for EAP
   include_tasks: pem_cert_eap.yml
-  when: (eduroam.EAP is defined) and (eduroam.EAP.certificate is defined)
+  when: eduroam.EAP is defined and eduroam.EAP.certificate is defined and eduroam.EAP.private_key is defined
```

Comment:
I'm not lazy to write parenthesis, I'm lazy/dumb to learn about operator priority. But accepting anyway.

Reply:
Thank you.